core/src/main/res/values/strings.xml

```diff
@@ -127,8 +127,10 @@
   <string name="tts_pause">pause</string>
   <string name="tts_resume">resume</string>
   <string name="stop">stop</string>
+  <string name="configure_storage_option_title">Download book to internal storage?</string>
```

Comment:
Is the string value appropriate?

Reply:
I suggest something like"Choose the storage location"

Reply:
if we choose "Choose the storage location" then the buttons having strings
will be wrong,
"configure" "yes"
 I followed what @kelson42 said in #2624 
 here->https://github.com/kiwix/kiwix-android/pull/2624#issuecomment-813025562
 I would sure change, but do look once at the comment :)

Reply:
Oh okay, my bad

Reply:
I'm sorry. My bad I forget that comment.

Reply:
its total fine :) @gouri-panda @4shutosh